docs/topics/db/models.txt

```diff
@@ -1415,3 +1415,47 @@ any model field in any ancestor model.
     :doc:`The Models Reference </ref/models/index>`
         Covers all the model related APIs including model fields, related
         objects, and ``QuerySet``.
+
+Organizing models
+=================
+
+The :djadmin:`manage.py startapp <startapp>` script creates a package structure
+for an application that includes a file called ``models.py``, but you may
+also define multiple files containing models, contained in a module. For example,
```

Comment:
I think the word for the-thing-those-multiple-files-are-contained-in is "package." (or maybe "sub-package"?)